### DISCUSSION TITLE
How does "enclosure" & "enclosure-thumbnail" is working ?

### DISCUSSION THREAD

**Discussion by ikus060** [FreshRSS/FreshRSS#6627] [Q&A]
Hello 

I'm trying to configure FreshRSS with XPath + HTML Web scraping and I'm running into issue. 

After configuring the best I can, the reader display the thumbnail after the full content. Is it possible to avoid that ?

Here is a snippet of the output I see:
```
<!-- FULLCONTENT start //-->
...
<!-- FULLCONTENT end //-->
<figure class="enclosure">
	<p class="enclosure-content">
		<img class="enclosure-thumbnail" src="my-thumbnail-url" alt="">
	</p>
</figure>
```

What I'm expecting is to display only the full content without adding the thumbnail after. Is that possible ?

Also, when updating the xpath expression, what is the step to reload or refresh according to my changes ? I'm not sure if I need to clear the cache, click reload article or click Update feeds as it all seams very similar to me.

Thanks for helping

**Comment by Frenzie** [comment]
> After configuring the best I can, the reader display the thumbnail after the full content. Is it possible to avoid that ?

If it's only delineated by comments I wouldn't get my hopes up, but you might be able to do something with `not()`.

> what is the step to reload or refresh according to my changes ?

Normally you'd want reload articles. Clear cache can't hurt but I don't think it should be necessary. Update feeds will only add (and process) new entries without affecting old ones.

**Comment by Alkarex** [comment]
Regarding thumbnails, duplicate of https://github.com/FreshRSS/FreshRSS/issues/4999#issuecomment-1777238211

There is no option to hide enclosures / thumbnails so far but this can be done from the Custom CSS extension with something like (adapt the feed ID to your needs):

```css
div[data-feed="123"] figure.enclosure {
	display: none;
}
```

Regarding updating the XPath expression, to supplement Frenzie's answer and depending on what you want, you may want to click on "Delete all articles", and then "Update feed".
